src/test/java/seedu/duke/tools/UITest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.duke.tools;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class UITest {
+    private final ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
+
+    @BeforeEach
+    public void setUp() {
+        System.setOut(new PrintStream(outputStreamCaptor));
+    }
+
+    @Test
+    void draw_main_separator() {
```

Comment:
Based on convention, perhaps a method name like 'drawSeparator_mainFeature_success' would be more descriptive?